tests/test_integration/test_cpp_regression.py

```diff
@@ -20,6 +20,9 @@
 import numpy as np
 import pytest
 
+
```

Comment:
I vote for removing these tests to resolve #312. I don't see too much value in these tests. @jdahm

Reply:
I agree. We have other tests that check the specific output of stencil calls to validate the correctness.